dlvm/gcp-gpu-utilization-metrics/create_gpu_metrics.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Create GPU metrics."""
+import os
+
+from google.cloud import monitoring_v3
+
+GPU_UTILIZATION_METRIC_NAME = 'gpu_utilization'
+GPU_MEMORY_UTILIZATION_METRIC_NAME = 'gpu_memory_utilization'
+
+
+class MissingProjectIdError(Exception):
+  pass
+
+
+def add_new_metrics(project_id, metric_type, desc):
+  """Add new Metrics for StackDriver.
```

Comment:
does the user need to activate stackdriver to use this?

Reply:
Not required